ext/cuda/remapping_distributed.jl

```diff
@@ -130,9 +130,7 @@ function set_interpolated_values_kernel!(
         out[i, j, k] = 0
         for t in 1:Nq
             out[i, j, k] +=
-                I[i, t] *
-                I[i, s] *
-                (
+                I[i, t] * (
```

Comment:
Was this a bug? Why aren't our unit tests catching this?

Reply:
Ok, good to know. Can we add a unit test in ClimaCore to exercise this?

Reply:
Sure, that's easy to add. I only discovered this bug last night so I'll add some tests today.

Reply:
There are unit tests for 1D extruded spaces, but they were skipped on GPU. There are https://github.com/CliMA/ClimaCore.jl/blob/a1c6470b8ea8f32143be190feea56e91749e9128/test/Remapping/distributed_remapping.jl#L46-L147.

Reply:
This test has been reenabled on GPUs and appears to be passing now.